main.go

```diff
@@ -197,6 +198,8 @@ func main() {
 
 	runner := NewDumpRunner(basePath)
 
-	log.Print("Running tasks...")
-	RunAllTasks(runner, basePath, *concurrentTasks)
+	log.Print("Running dump tasks...")
+	RunAllDumpTasks(runner, basePath, *concurrentTasks)
```

Comment:
`flag.Int` & family return pointer values.
"If you like, you can bind the flag to a variable using the Var() functions." ([pkg/flag docs](https://golang.org/pkg/flag/))

Because `concurrentTasks` is a `*int`, we need to dereference because we want an `int` type in the argument.

`flag.IntVar` takes a pointer because it needs to mutate the value. And `flag.Int` returns a pointer for similar reasons, otherwise you'd need to have the flag value determined at the time you declare the flag, which doesn't make sense.